### PR TITLE
feat: Added quiz category page

### PR DESCRIPTION
**Pull request by krishanu2911**
Added Categories page of the quiz app responsive with help of grids.

https://user-images.githubusercontent.com/75560307/153724684-60fe061e-f35a-4090-a259-dd6e6322141d.mp4

### CODE REVIEW

File: CSS/quiz-category.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+.nav-bar{
+    padding: 0;
+    grid-area: nav-bar;
+    background-color: var(--section-background-color);
+}
+.list-pills{
+    width: fit-content;
+    background-color: var(--background-color);
+}
+.hide{
```
Comment (tanishq20) [inline CSS/quiz-category.css]: You can shift this to the utility folder.

**Review by rahulcs754 — APPROVED** [review]
i liked your quiz  design
code looks clean.

**Review by tanishq20 — COMMENTED** [review]
Suggested small change.

**Review by nakulsharma15 — APPROVED** [review]
Looks good. No changes suggested

File: CSS/quiz-category.css (hunk 1 above)
Comment (krishanu2911) [reply CSS/quiz-category.css]: Thanks for pointing out added the util file and added hide class

**Review by tanishq20 — APPROVED** [review]

**Pull request merged**